frame/support/src/lib.rs

```diff
@@ -1855,6 +1855,22 @@ pub mod pallet_prelude {
 /// pub(super) type MyStorage<T> = StorageValue<Value = u32>;
 /// ```
 ///
+/// The optional attribute `#[pallet::whitelist_storage]` will declare the
+/// storage as whitelisted from benchmarking. Doing so will exclude reads of
+/// that value's storage key from counting towards weight calculations during
+/// benchmarking.
+///
+/// This attribute should only be attached to storages that are known to be
+/// read/used in every block. This will result in a more efficient read pattern
+/// for the value and a more accurate benchmarking weight.
```

Comment:
How is the read pattern more efficient? I'm not aware of any optimizations that happening because of this.

Reply:
You're right that feature wasn't included in this PR. Fixing